### PR TITLE
feat: upgrade APISIX to 2.10.2

### PR DESCRIPTION
**Pull request by tzssangglass**
Signed-off-by: tzssangglass <tzssangglass@gmail.com>

### CODE REVIEW

**Comment by tzssangglass** [comment]
wait for rpm ready

**Review by tokers — APPROVED** [review]

File: centos/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM centos:7
 
-ARG APISIX_VERSION=2.10.1.1
+ARG APISIX_VERSION=2.10.2.1
```
Comment (spacewander) [inline centos/Dockerfile:3]:
```suggestion
ARG APISIX_VERSION=2.10.2
```

**Review by lilien1010 — APPROVED** [review]
LGTM

**Review by tao12345666333 — APPROVED** [review]

**Review by spacewander — CHANGES_REQUESTED** [review]
Wait for rpm.

**Review by spacewander — APPROVED** [review]

**Pull request merged**